docs-new/docs/graphql/core/api-reference/relay-graphql-api/query.mdx

```diff
@@ -116,19 +112,12 @@ connection-object {
 | pageInfo | [PageInfo](#pageinfo)! |
 | edges    | \[[Edge](#edge)!\]!    |
 
-<div class="note">
-
-<div class="title">
-
-Note
-
-</div>
+:::info Note
 
 For more details on the Relay `connection` object type, refer to the
-[Relay
-docs](https://relay.dev/graphql/connections.htm#sec-Connection-Types).
+[Relay docs](https://relay.dev/graphql/connections.htm#sec-Connection-Types).
 
-</div>
+:::
 
 ### PageInfo
```

Comment:
missed id here and check for remaining sections too

Reply:
Done